obal/data/modules/copr_project.py

```diff
@@ -0,0 +1,79 @@
+"""
+Create a Project in Copr
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.copr import copr_cli, CoprCliCommandError, full_name # pylint:disable=import-error,no-name-in-module
+
+
+def project_exists(user, project, module):
+    """
+    Return true if a project already exists for a user
+    """
+    command = [
+        'list',
+        user
+    ]
+
+    try:
+        project_list = copr_cli(command)
+    except CoprCliCommandError as error:
+        module.fail_json(msg='Copr project listing failed', command=' '.join(error.command), output=error.message)
+
+    return "Name: {}".format(project) in project_list
```

Comment:
This will be `True` if there is a project called `foreman-nightly` but the value of the `project` variable is just `foreman`.

I don't think it's a real-world issue (for our usage), as we have rather unique names for all our projects, but still IMHO something we should cleanup (at some point)

Reply:
I could opt to do this instead with a regex so that a line terminator is included?

Reply:
Or you split the lines (`.splitlines()`) and strip them like this:
https://github.com/theforeman/obal/blob/60e8b53f1993e961f9f26a922633194648a0184a/tests/fixtures/mockbin/mockbin#L43

Then you can use the `in` on the resulting list?

Either way works